app/src/main/java/mozilla/lockbox/support/Constant.kt

```diff
@@ -27,7 +27,17 @@ object Constant {
     }
 
     object Faq {
-        const val uri = "https://lockbox.firefox.com/faq.html#top"
+        const val uri = "https://lockbox.firefox.com/faq.html"
```

Comment:
Who localizes this? i.e. should this come from `resources`?

Reply:
Good question; I have no idea what the l10n story is for our FAQ pages. maybe @devinreams knows more?

Reply:
Thanks for bringing it up. Long term the pages/content will move to SUMO and will add this to our localization infrastructure todo list/questions.

For now, and I think you all agree, I don't think we need to try to future-proof or handle it quite yet...